woocommerce-stats-by-currency.php

```diff
@@ -133,3 +133,30 @@ function wsbc_add_stats() {
 
 }
 add_action( 'woocommerce_after_dashboard_status_widget', 'wsbc_add_stats' );
+
+/**
+ * Check required plugins
+ *
+ * @return void
+ */
+
+function wsbc_required_plugins() {
+	$activated = apply_filters( 'active_plugins', (array) get_option( 'active_plugins', array() ) );
+	$required  = array(
+		'woocommerce/woocommerce.php',
```

Comment:
No need to check for `woocommerce` as `woocommerce-product-price-based-on-countries` will have it as a dependency.